tahoe_sites/helpers.py

```diff
@@ -104,3 +105,41 @@ def is_active_admin_on_organization(user, organization):
         is_active=True,
         is_admin=True,
     ).exists()
+
+
+def create_tahoe_site(organization, site):
+    """
+    Link a site to an organization. Together they are a tahoe-site
+
+    :param organization: Organization object to be linked
+    :param site: Site object to be linked
+    :return: TahoeSite object
+    """
+    if zd_helpers.should_site_use_org_models():
+        organization.sites.add(site)
+        return None
+    return TahoeSite.objects.create(organization=organization, site=site)
+
+
+def get_site_by_organization(organization):
+    """
+    Get the site by its related organization
+
+    :param organization: Organization object to filter on
+    :return: Site object related to the given organization
+    """
+    if zd_helpers.should_site_use_org_models():
+        return organization.sites.first()
```

Comment:
Should get either one `Site` or throw an exception due to malformed database entry.

```suggestion
        return organization.sites.get()
```

Reply:
done